docs/src/how-to/install/configuration-options.rst

```diff
@@ -576,3 +576,333 @@ Second, also set the option under the `account-pages` section:
+| Yes, `tA`                          | Yes, another team tB            | true                               | Irrelevant                               | Irrelevant                                | Not found                        | Not found                            |
++------------------------------------+---------------------------------+------------------------------------+------------------------------------------+-------------------------------------------+----------------------------------+--------------------------------------+
+| Yes, `tA`                          | Yes, another team tB            | false                              | `SearchVisibilityNoNameOutsideTeam`      | Irrelevant                                | Found                            | Not found                            |
++------------------------------------+---------------------------------+------------------------------------+------------------------------------------+-------------------------------------------+----------------------------------+--------------------------------------+
+| Yes, `tA`                          | No                              | false                              | `SearchVisibilityNoNameOutsideTeam`      | There’s no team B                         | Found                            | Not found                            |
++------------------------------------+---------------------------------+------------------------------------+------------------------------------------+-------------------------------------------+----------------------------------+--------------------------------------+
+
+Another way to express this is with the pseudocode:
+
+.. code:: js
+
+  def search(searcher: User, query: String) {
+    val fullTextResults = searchFullText(query)
+    val exactResults = findExactHandle(query)
+  
+    if(searcher.team == null) {
+      return exactResults + fullTextResults.filter { foundUser => foundUser.team == null }
+    } 
+  
+    // else: searcher team is NOT null
+    if (backend.searchSameTeamOnly) {
+      return exactResults.filter { foundUser => foundUser.team == searcher.team } + fullTextResults.filter { foundUser => foundUser.team == searcher.team}
+    }
+  
+    // else: backend config is NOT searchSameTeamOnly
+    if (searcher.team.TeamSearchVisibility == SearchVisibilityNoNameOutsideTeam) {
+      return exactSearchResults + fullTextResults.filter { foundUser => foundUser.team == searcher.team }  
+    }
+  
+    // else: searcher.team.TeamSearchVisibility == SearchVisibilityStandard
+    return exactSearchResult + fullTextResults.filter { foundUser => foundUser.team == null || (foundUser.team.SearchVisibilityInbound == SearchableByAllTeams || foundUser.team == searcher.team) }
+  }
```

Comment:
I'd remove this pseudocode. I can't guarantee that it's fully up-to-date.

Reply:
removed. thanks